<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPopup</class>
 <widget class="QWidget" name="LoginPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Log In</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #555879
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>700</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 32px
}</string>
   </property>
   <property name="text">
    <string>Log In</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>401</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="usernameField">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
  color: white;
  border: 2px solid #DED3C4;
  border-radius: 10px;
  padding: 10px 25px;
  background: transparent;
}

QLineEdit:focus {
  border: 2px solid #F4EBD3;
}</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhLatinOnly</set>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="passwordField">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
  color: white;
  border: 2px solid #DED3C4;
  border-radius: 10px;
  padding: 10px 25px;
  background: transparent;
}

QLineEdit:focus {
  border: 2px solid #F4EBD3;
}</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhHiddenText|Qt::ImhLatinOnly|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Password</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showPwdCB">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator {
        width: 0;
        height: 0;
    }
    QCheckBox {
        spacing: 0;
        margin: 0;
    }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="loginBtn">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  color: #555879;
  padding: 0.2em 1.7em;
  font-size: 14px;
  border-radius: 0.5em;
  background: #F4EBD3;
  border: 1px solid #DED3C4;
}

QPushButton:hover {
  background: rgb(230, 221, 209);
}

QPushButton:pressed {
  color: #98A1BC;
  background: #DED3C4;
}
</string>
      </property>
      <property name="text">
       <string>Log In</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="createAccountBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  color: #F4EBD3;
  padding: 0.4em 1.2em;
  font-size: 14px;
  border-radius: 0.5em;
  background: #555879;
  border: 1px solid #F4EBD3;
}

QPushButton:hover {
  color: rgb(230, 221, 209);
  border: 1px solid rgb(230, 221, 209);
}

QPushButton:pressed {
  color: #DED3C4;
  border: 1px solid #DED3C4;
}
</string>
   </property>
   <property name="text">
    <string>Create Account</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
